metrics/counter_float_64_test.go

```diff
@@ -10,6 +13,20 @@ func BenchmarkCounterFloat64(b *testing.B) {
 	}
 }
 
+func BenchmarkCounterFloat64Parallel(b *testing.B) {
+	c := NewCounterFloat64()
+	b.ResetTimer()
+	var wg sync.WaitGroup
+	for i := 0; i < b.N; i++ {
+		wg.Add(1)
+		go func(i float64) {
+			c.Inc(i)
+			wg.Done()
+		}(float64(i))
+	}
+	wg.Wait()
+}
```

Comment:
I think this benchmark is a bit too parallel -- I suspect there's quite some noise from just maintaining `b.N` goroutines.

Perhaps a more realistic benchmark wold be do spin up 10 routines, and all ten routines does `b.N` calls to `c.Inc`.